example/lib/main.dart

```diff
@@ -174,66 +164,51 @@ class _MyAppState extends State<MyApp> {
-                    //     onPressed: () async {
-                    //       print('loading..');
-                    //       var response;
-                    //       try {
-                    //         response =
-                    //         await FlutterPericulum.statementTransaction(token:
-                    //         "${dotenv.env['tokenKey']}", key: "125").then((value) => {
-                    //          print('loading completed..'),
-                    //           debugPrint(value.toString()),
-                    //         });
-                    //       } on PlatformException catch (e) {
-                    //         if (kDebugMode) {
-                    //           print(e);
-                    //         }
-                    //         } on Exception catch (e) {
-                    //         if (kDebugMode) {
-                    //           print(e);
-                    //         }
-                    //       }
-                    //     },
-                    //     child: const Text(
-                    //       'Get Exisiting credit score',
-                    //     )),
+                    TextButton(
+                        onPressed: () async {
+                          setState(() {
+                            isLoading = true;
+                          });
+                          var response;
+                          try {
+                            response =
+                                await FlutterPericulum.getExisitingCreditScore(
+                                        token: "${dotenv.env['tokenKey']}",
+                                        statementKey: "123")
+                                    .then((value) => {
+                                          setState(() {
+                                            isLoading = false;
+                                            creditResponse = value;
+                                            debugPrint(creditResponse[0]
+                                                .baseScore
+                                                .toString());
+                                          }),
+                                        });
+                          } on PlatformException {
+                            setState(() {
+                              isLoading = false;
+                            });
+                          } on Exception catch (e) {
+                            isLoading = false;
+                            if (kDebugMode) {
+                              print(e);
+                            }
+                            setState(() {
+                              isLoading = false;
+                            });
+                          }
+                        },
+                        child: const Text(
+                          'Get Transaction analytics',
+                        )),
+                    InkWell(
```

Comment:
Just a little confused about this. Is this somethin in the example app that just triggers the getStatementTransactions?

Reply:
This just meant for test

Reply:
this isn't part of the sdk